pages/[...slug].vue

```diff
@@ -1,12 +1,34 @@
 <template>
   <div>
-    <Breadcrumbs v-if="page.breadcrumbs" />
-    <DrupalTabs v-if="page.local_tasks" :tabs="page.local_tasks" />
-    <component :is="renderCustomElements(page.content)" />
+    <NuxtLayout :name="layout">
+      <main>
+        <SiteBreadcrumbs />
+        <DrupalTabs v-if="page.local_tasks" :tabs="page.local_tasks" />
+        <component :is="renderCustomElements(page.content)" />
+      </main>
+    </NuxtLayout>
   </div>
 </template>
 
 <script lang="ts" setup>
-const { fetchPage, renderCustomElements } = useDrupalCe()
-const page = await fetchPage(useRoute().path, { query: useRoute().query })
+const { fetchPage, renderCustomElements } = useDrupalCe();
+const page = await fetchPage(useRoute().path, { query: useRoute().query });
+// Set to false to support custom layouts, using <NuxtLayout> instead.
```

Comment:
not sure what this refers to, the next boolean is false?

Reply:
by setting layout to false, it basically uses nuxt-layout component instead, and tells nuxt not to use its automatic layout system.